src/components/MobileMenu.tsx

```
@@ -0,0 +1,122 @@
+        <div className='flex flex-col h-full'>
+          {/* Header */}
+          <div className='flex items-center justify-between p-4 border-b border-gray-200'>
+            <div className='flex items-center space-x-2'>
+              <PenTool className='h-5 w-5 text-primary' />
+              <span className='font-bold text-lg text-dark'>Paperlyte</span>
+            </div>
+            <button
+              onClick={onClose}
+              className='p-2 rounded-lg hover:bg-gray-100 transition-colors'
+              aria-label='Close menu'
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)